plugins/field-grid-dropdown/src/index.ts

```diff
@@ -9,169 +9,215 @@
  * @author kozbial@google.com (Monica Kozbial)
  */
 
-import Blockly from 'blockly/core';
+import * as Blockly from 'blockly/core';
+
+/**
+ * A config object for defining a field grid dropdown.
+ */
+export interface FieldGridDropdownConfig extends Blockly.FieldConfig {
```

Comment:
I /think/ this should extend the `FieldDropdownConfig` instead, because we are passing the config to `super`. So if we ever define more config options for the `FieldDropdown` those will be supported by the `FieldGridDropdown`

Reply:
Good catch!